var/spack/repos/builtin/packages/paraver/package.py

```diff
@@ -7,24 +7,61 @@
 import os
 
 
-class Paraver(Package):
+class Paraver(AutotoolsPackage):
```

Comment:
At some point we should rename this package to `wxparaver`. This will require creating a new package called `wxparaver` and marking all of the versions in this package as `deprecated=True`. You can actually do that if you want in this PR, or wait for a future PR.

Reply:
done